app/src/main/java/io/transcend/samplesdk/ManageConsentPreferences.java

```diff
@@ -40,8 +45,10 @@ private void setUpButtons() {
         button.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                TranscendWebView transcendWebView = (TranscendWebView) findViewById(R.id.transcendWebView);
-                transcendWebView.showWebView(null);
+                webViewContainer.removeAllViews();
+                TranscendWebView transcendWebView = new TranscendWebView(context, "https://transcend-cdn.com/cm-test/ee571c7f-030a-41b2-affa-70df8a47b57b/airgap.js", (TranscendListener.OnCloseListener) () -> webViewContainer.setVisibility(View.GONE));
+                webViewContainer.addView(transcendWebView);
+                webViewContainer.setVisibility(View.VISIBLE);
```

Comment:
kind of funky.. so we're removing the view and re-loading it every time the button is clicked? How come it's not possible to just keep the view around but toggle visibility?

Reply:
Seems like we can't reuse views and i should remove the `destroyOnClose` property ?